_profiles/Study/0.2-DRAFT.html

```diff
@@ -0,0 +1,405 @@
+---
+redirect_from:
+- "devSpecs/Study/specification"
+- "devSpecs/Study/specification/"
+- "/devSpecs/Study/"
+- "/specifications/drafts/Study"
+- "/profiles/Study/"
+
+name: Study
+
+previous_version: 0.1-DRAFT-2018_11_15
+previous_release:
+
+status: revision
+spec_type: Profile
+group: studies
+use_cases_url: /useCases/Study/
+cross_walk_url: https://docs.google.com/spreadsheets/d/1320sC6scXagwB5kJyGObuW5YkWGrZ1JNIYk-rmC3pyE/
+gh_tasks: https://github.com/Bioschemas/bioschemas/labels/type%3A%20Study
+live_deploy: ''
+
+parent_type: Study
+hierarchy:
+- Thing
+- CreativeWork
```

Comment:
Being this the Study type profile, should not the hierarchy be Thing-CreativeWork-Study?